shuntingYard/main.c

```diff
@@ -0,0 +1,96 @@
+#include "shuntingYard.h"
+
+#include<stdlib.h>
```

Comment:
Надо пробел после include:
```suggestion
#include <stdlib.h>
```